scripts/screens/MakeClanScreen.py

```diff
@@ -1309,7 +1309,7 @@ def random_biome_selection(self):
     def _get_cat_tooltip_string(self, cat: Cat):
         """Get tooltip for cat. Tooltip displays name, sex, age group, and trait."""
 
-        return f"<b>{cat.name}</b><br>{cat.genderalign}<br>{cat.age}<br>{cat.personality.trait}"
+        return f"<b>{cat.name}</b><br>{cat.genderalign}<br>{cat.age.value}<br>{cat.personality.trait}"
```

Comment:
Requesting `cat.age` in a string will automatically request the `value` as far as I know.
```suggestion
        return f"<b>{cat.name}</b><br>{cat.genderalign}<br>{cat.age}<br>{cat.personality.trait}"
```